.gitignore

```diff
@@ -1 +1,2 @@
+echo coverage >> .gitignore
```

Comment:
Please delete this, this is a line containing a console command that doesn't do anything in this file. My bad.